host/linux/sgxioctl.c

```diff
@@ -50,14 +51,19 @@ OE_PACK_END
 int sgx_ioctl_enclave_create(int dev, sgx_secs_t* secs)
 {
     SGXECreateParam param;
+    int rc = 0;
 
     if (dev == -1 || !secs)
         return -1;
 
     memset(&param, 0, sizeof(param));
     param.secs = (uint64_t)secs;
 
-    return ioctl(dev, SGX_IOC_ENCLAVE_CREATE, &param);
+    rc = ioctl(dev, SGX_IOC_ENCLAVE_CREATE, &param);
```

Comment:
Compared to the messages later in this PR, the ones with rc and with mbedtls seem too terse.

Reply:
ioctl is a very generic call, which does not reveal too much operation specific error code.
It's up to the target (intel's driver in this case) to return more meaningful error code.